react/CategoryBlock.tsx

```diff
@@ -89,7 +90,7 @@ const CategoryBlock: FunctionComponent<WrappedComponentProps & any> = ({
       action = success
         ? {
             label: intl.formatMessage(messages.seeCart),
-            href: '/checkout/#/cart',
+            href: checkoutUrl ?? '/checkout/#/cart',
```

Comment:
Same thing, if someone sends an empty string as a prop, this would break the redirect

Reply:
Done!